src/server.cpp

```diff
@@ -6953,6 +6953,18 @@ void loadDataFromDisk(void) {
 
     if (g_pserver->m_pstorageFactory)
     {
+        if (g_pserver->config_notify_flash_load) {
+            moduleFireServerEvent(REDISMODULE_EVENT_LOADING, REDISMODULE_SUBEVENT_LOADING_FLASH_START, NULL);
+            for (int idb = 0; idb < cserver.dbnum; ++idb) {
+                auto spsnapshot = g_pserver->db[idb]->CloneStorageCache();
+                spsnapshot->enumerate([idb](const char *rgchKey, size_t cchKey, const void *, size_t) -> bool {
```

Comment:
we already enumerate when setting up the StorageCache, can you do this there so we don't double startup time?